components/sidebar_toggle.py

```diff
@@ -50,7 +50,17 @@ def add_callback(app):
         def _toggle_sidebar(n_clicks):
             SidebarToggle.is_open = not SidebarToggle.is_open
             return (
-                {"display": "block"} if SidebarToggle.is_open else {"display": "none"},
+                {
+                    "display": "block",
+                    "height": "inherit",
```

Comment:
These height and inherit stylings were moved in a3e7a1c. Those changes seem to have been lost in the merge.